en/mkdocs.yml

```diff
@@ -78,19 +78,17 @@ nav:
       - 'Customizing the Management Console': setup/customizing-the-Management-Console.md
       - 'Shutting Down and Restarting the Server': setup/shutting-Down-and-Restarting-the-Server.md
     - 'Deployment Guide':
-      - 'Deployment Patterns':
+      - 'Deployment Guide': setup/deploying-the-identity-server.md
+      - 'Deployment Patterns': 
         - 'Deployment Patterns': setup/deployment-patterns.md
-        - 'Setting Up Deployment Pattern 1': setup/setting-up-deployment-pattern-1.md
-        - 'Setting Up Deployment Pattern 2': setup/setting-Up-Deployment-Pattern-2.md
-        - 'Setting Up Separate Databases for Clustering': setup/setting-Up-Separate-Databases-for-Clustering.md
-      - 'Deployment Guidelines in Production': setup/deployment-Guidelines-in-Production.md
+        - 'Fronting with the Nginx load balancer': setup/fronting-with-the-nginx-load-balancer.md
+        - 'Configuring the Datasource' :setup/configuring-the-datasources.md
+        - 'Setting Up Separate Databases for Clustering': setup/setting-up-separate-databases-for-clustering.md
+      - 'Deployment Guidelines in Production': setup/deployment-suidelines-in-Production.md
```

Comment:
"suidelines" should be changed to "guidelines" in this folder path

Reply:
Fixed